kaggle_survey_clean_write.Rmd

```diff
@@ -59,7 +64,8 @@ df_basic<- df %>%
   mutate(spend_ml_cloud_work_USD = str_remove_all(spend_ml_cloud_work_USD,"\\(USD\\)")) %>%
   mutate(code_exp_years= str_remove_all(code_exp_years, "years")) %>%
   mutate(code_exp_years= str_replace_all(code_exp_years, "I have never written code", "None"))
-df_basic
+df_basic$id <- 1:nrow(df_basic)
```

Comment:
Does this make the index? Can it be done when the data is loaded to reduce repetition? I don't actually know the answer to that question, just wanted to know how it works.

Reply:
yes, this is making the index. its possible to do it at the beginning, but then when you slice out the individual dataframes you would need to make sure you are keeping the id column.